### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by ameliorater**
Fixed typos and some small errors in math I noticed while reading through the Sets, Logic, Computation textbook build.

### CODE REVIEW

File: content/first-order-logic/completeness/compactness.tex [hunk 1]
```diff
@@ -149,7 +149,7 @@
 \tagprob{FOL}
 \begin{prob}
 In the standard model of arithmetic~$\Struct{N}$, there is no
-!!{element}~$k \in \Domain{N}$ which satisfies every formula~$\num{n}
+!!{element}~$n \in \Domain{N}$ which satisfies every formula~$\num{n}
```
Comment (rzach) [inline content/first-order-logic/completeness/compactness.tex:152]: k is actually correct. A k which satisfies all \num{n} < x would have to be larger than all n. No such k exists in the standard model. Can you change that back and push?

Comment (ameliorater) [reply content/first-order-logic/completeness/compactness.tex]: Yep, just changed it back and pushed! Thanks

Comment (rzach) [reply content/first-order-logic/completeness/compactness.tex]: No thank you!

**Comment by rzach** [comment]
Thank you!

**Pull request merged**